addons/udes_ldap/models/res_company_ldap.py

```diff
@@ -0,0 +1,54 @@
+"""
+UDES LDAP configuration model.
+
+In this model we override the standard authentication approach by using the
+credentials of the logging-in user to search for themselves on the LDAP
+server.  This removes the need for a service account to perform the search.
+"""
+
+from ldap.dn import escape_dn_chars
+from odoo import models, fields
+
+
+class CompanyLDAP(models.Model):
+    """LDAP configuration model."""
+
+    _inherit = "res.company.ldap"
+
+    u_ldap_binddn_fmt = fields.Char(
+        string="LDAP binddn format",
```

Comment:
Does this string need to be capitalised: "LDAP Binddn Format"? I understand it is going to be invisible in the view, but maybe just for consistency?

Reply:
I _think_ the logic behind the casing on this model seems to be rather inconsistent, for example "LDAP Server address", "LDAP base", "Create User", so I'm not sure what's best here.  I'm inclined to leave as-is, but I'm open to persuasion.

Reply:
Ideally all strings should be properly capitalised, so would be good to do it for new fields even though the old ones are not consistent with it.

Reply:
Persuaded :smile:  and fixed.